### DISCUSSION TITLE
Raspberry looses connection after final reboot

### DISCUSSION THREAD

**Discussion by xavi-developer** [pivpn/pivpn#1677] [Support]
Hello,

I had pivpn working properly in a Raspberry Pi OS Lite (2022-09-22) until some days ago when it suddenly stopped working. I have reinstalled everything. While I was doing the installation I was pinging [google.com](https://google.com/) properly. Also after completing the installation (no reboot) ping worked properly. After the final reboot, I have lost ssh access (connection refused) and pinging [google.com](https://google.com/) results in "ping: google.com: Temporary failure in name resolution"

I have installed it 3 times already ( curl -L https://install.pivpn.io | bash) with the same result (2023-01-16). No special configurations: OpenVPN, Google NS, static IP, Public IP.

Am I missing something?

Thanks

**Comment by coolapso** [comment]
please provide the details requested here: https://raw.githubusercontent.com/pivpn/pivpn/master/.github/ISSUE_TEMPLATE/pivpn-report.md
